### PR TITLE
- implement support for many-to-many relations

### PR DESCRIPTION
**Pull request by Insolita**
- update docs
- fix "SET NOT NULL"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,19 +95,27 @@ You may specify custom PHP code for generating fake data for a property:
 
 Specify the table name for a Schema that defines a model which is stored in the database.
 
+### `x-pk`
+
+Explicitly specify primary key name for table, if it different from "id"
```
Comment (cebe) [inline README.md:100]:
```suggestion
Explicitly specify primary key name for table, if it is different from "id" 
```

File: src/generator/default/dbmodel.php [hunk 2]
```diff
@@ -47,11 +51,24 @@ public function get<?= $relation->getCamelName() ?>()
...
         return $this-><?= $relation->getMethod() ?>(\<?= trim($relationNamespace, '\\') ?>\<?= $relation->getClassName() ?>::class, <?php
             echo str_replace(
-                    [',', '=>', ', ]'],
-                    [', ', ' => ', ']'],
-                    preg_replace('~\s+~', '', VarDumper::export($relation->getLink()))
-            )
+    [',', '=>', ', ]'],
+    [', ', ' => ', ']'],
+    preg_replace('~\s+~', '', VarDumper::export($relation->getLink()))
+)
```
Comment (cebe) [inline src/generator/default/dbmodel.php:57]: bug in style fixer or is this change on purpose?

File: tests/specs/blog_v2/migrations/m200000_000004_create_table_posts2tags.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ */
+class m200000_000004_create_table_posts2tags extends \yii\db\Migration
+{
+    public function up()
+    {
+        $this->createTable('{{%posts2tags}}', [
+            'id' => $this->bigPrimaryKey(),
+            'post_id' => $this->bigInteger()->notNull(),
+            'tag_id' => $this->bigInteger()->notNull(),
+        ]);
```
Comment (cebe) [inline tests/specs/blog_v2/migrations/m200000_000004_create_table_posts2tags.php:14]: junction table primary key should be composite PK: `PRIMARY KEY(post_id, tag_id)` `id` does not make sense here.

**Review by cebe — CHANGES_REQUESTED** [review]

File: tests/specs/blog_v2/migrations/m200000_000004_create_table_posts2tags.php (hunk 3 above)
Comment (Insolita) [reply tests/specs/blog_v2/migrations/m200000_000004_create_table_posts2tags.php:13]: done

File: src/generator/default/dbmodel.php (hunk 2 above)
Comment (Insolita) [reply src/generator/default/dbmodel.php:57]: style-fixer

**Review by cebe — APPROVED** [review]

**Comment by cebe** [comment]
Great work, thank you!
